k8s/helm/templates/frontend-server-service.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: frontend-server
+  namespace: chroma
+spec:
+  ports:
+    - name: frontend-server
+      port: 8000
+      targetPort: 8000
+  selector:
+    app: frontend-server
+  type: LoadBalancer
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend-server
+  namespace: chroma
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: frontend-server
+  template:
+    metadata:
+      labels:
+        app: frontend-server
+    spec:
+      containers:
+        - name: frontend-server
+          image: "{{ .Values.imageUrl }}"
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 8000
+          volumeMounts:
+            - name: chroma
+              mountPath: /test
```

Comment:
is this the data dir (e.g. `/chroma/chroma`)? Alternatively, we can specify env var `PERSIST_DIRECTORY=/test`